projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/structlog_plugin.py

```diff
@@ -77,7 +81,146 @@ def process(self, msg, kwargs):
 """
 
 
+class StructlogConfig:
```

Comment:
I realy think the structlog_plugin module is very long. 

It is the plugin's entry point (basically similar to "main" method for an application), so it should only bootstrap things and delegate most of the logic to other modules. 

So please let's please StructlogConfig in separate module (aka file). 

I'd refactor StructlogPlugin to be shorter as well. But that's out of scope here.

Reply:
I can do it in a separate PR if that's ok.